### DISCUSSION TITLE
Cant get serverless functions to work in Python

### DISCUSSION THREAD

**Discussion by mwt** [vercel/vercel#6921] [Help] [answered]
I am trying to get a Python serverless function to work. However, even the examples in the documentation do not seem to work. When I go to the api endpoint, it just downloads the script without executing. You can see this behavior here:

https://econ-ipsum.vercel.app/api

The Vercel panel shows the function as valid:
![image](https://user-images.githubusercontent.com/13644285/139706809-cdaceddc-a640-48a7-a93c-cd60f0af9f92.png)

I have tried to rebuild the project, but it still does not work. Does anyone know what might be happening here? The code for the api is [here](https://github.com/mwt/econ-ipsum/blob/vercel/api/index.py). It has no dependencies.

**Comment by leerob** [comment, marked as answer]
Hey there – I know this is an older discussion, but I wanted to follow up here. If anyone else lands here and you're seeing issues, please let me know and happy to help out!

https://vercel.com/templates/python
